example_config.json

```diff
@@ -5,7 +5,7 @@
     "guild_id": 111111111111111111,
     "info_guild_id": "Guild (server) ID. Bot will only work in this guild.",
     "ticket_users": [508391840525975553],
-    "info_ticket_users": "User IDs. The bot will create threads under channels created by those users. The currently inputed ID is for the main Tickets bot",
```

Comment:
Maybe a note on the source of the ID provided should be kept?

Reply:
If we're making this a multi-use bot maybe we should keep arrays empty and have the bot provide user feedback (via console or other means, such as through Discord) as to the lack of configuration?

Reply:
Sorry I should probably be more clear, if this is a bot for servers other than ours, there needs to be more error checking and a broader functionality set to accomodate as such

Reply:
> Sorry I should probably be more clear, if this is a bot for servers other than ours, there needs to be more error checking and a broader functionality set to [accommodate] as such

I feel that they should understand via the comments in the code that they are missing some config info, but some error checking could be a nice general "lets not explode the bot by erroring out" catch